bus-mapping/src/exec_trace.rs

```diff
@@ -201,37 +213,43 @@ mod trace_tests {
                 EvmWord(BigUint::from(0x40u8)),
                 EvmWord(BigUint::from(0x80u8)),
             ],
-            Instruction::new(OpcodeId::PUSH1, Some(EvmWord(BigUint::from(0x80u8)))),
+            Instruction::new(
+                OpcodeId::PUSH1,
+                Some(EvmWord(BigUint::from(0x80u8))),
+            ),
             ProgramCounter::from(1),
             GlobalCounter::from(2),
         );
 
-        // Add StackOp associated to this opcode to the container & step.bus_mapping
-        step_2
-            .bus_mapping_instances_mut()
-            .push(container.insert(StackOp::new(
+        // Add StackOp associated to this opcode to the container &
+        // step.bus_mapping
+        step_2.bus_mapping_instances_mut().push(container.insert(
+            StackOp::new(
                 RW::WRITE,
                 GlobalCounter(3usize),
                 StackAddress::from(1022),
                 EvmWord(BigUint::from(0x80u8)),
-            )));
+            ),
+        ));
         let expected_exec_trace = ExecutionTrace {
             entries: vec![step_1, step_2],
             block_ctants: block_ctants.clone(),
             container,
         };
 
         // Obtained trace computation
-        let trace_loaded: Vec<ExecutionStep> =
-            serde_json::from_str::<Vec<ParsedExecutionStep>>(input_trace)
-                .expect("Error on parsing")
-                .iter()
-                .enumerate()
-                .map(|(idx, step)| ExecutionStep::try_from((step, GlobalCounter(idx))))
-                .collect::<Result<Vec<ExecutionStep>, Error>>()
-                .expect("Error on conversion");
-
-        let obtained_exec_trace = ExecutionTrace::new(trace_loaded, block_ctants);
+        let trace_loaded: Vec<ExecutionStep> = serde_json::from_str::<
```

Comment:
Remove the type annotation fixes the weird indention.
```suggestion
        let trace_loaded = serde_json::from_str::<
```